src/components/secrets/Secret.tsx

```diff
@@ -729,6 +729,7 @@ export const SecretForm: React.FC<SecretFormProps> = function (props) {
                 <div className="form-row__select-external-type flex">
                     <ReactSelect
                         placeholder="Select Secret Type"
+                        data-testid="secrets-data-type-select-dropdown"
```

Comment:
Remove as it won't have any affect. Use classNamePrefix instead.

Reply:
Done